site/content/en/docs/tasks/run_cronjobs.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Run A CronJob"
+date: 2023-12-12
+weight: 5
+description: >
+  Run a CronJob in a Kubernetes cluster with Kueue enabled.
+---
+
+This page shows you how to run a CronJob in a Kubernetes cluster with Kueue enabled.
+
+The intended audience for this page are [batch users](/docs/tasks#batch-user).
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+- [Kueue is installed](/docs/installation).
+- The cluster has [quotas configured](/docs/tasks/administer_cluster_quotas).
+
+## 0. Identify the queues available in your namespace
+
+Run the following command to list the `LocalQueues` available in your namespace.
+
+```shell
+kubectl -n default get localqueues
+# Or use the 'queues' alias.
+kubectl -n default get queues
+```
+
+The output is similar to the following:
+
+```bash
+NAME         CLUSTERQUEUE    PENDING WORKLOADS
+user-queue   cluster-queue   3
+```
+
+The [ClusterQueue](/docs/concepts/cluster_queue) defines the quotas for the
+Queue.
+
+## 1. Define the Job
+
+Running a CronJob in Kueue is similar to [running a CronJob in a Kubernetes cluster](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/)
+without Kueue. However, you must consider the following differences:
+
+- You should set the JobTemplate in CronJob in a [suspended state](https://kubernetes.io/docs/concepts/workloads/controllers/job/#suspending-a-job),
+  as Kueue will decide when it's the best time to start the Job.
+- You have to set the Queue you want to submit the Job to. Use the
+ `kueue.x-k8s.io/queue-name` label in `jobTemplate.metadata`
+- You should include the resource requests for each Job Pod.
```

Comment:
Because each of the job runs are queued, it's also important to consider what to do with https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#starting-deadline and https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#concurrency-policy

Kueue will work regardless, but users should be thinking about those options.